nuxeo-coldstorage/src/test/java/org/nuxeo/coldstorage/helpers/TestColdStorage.java

```diff
@@ -155,7 +171,8 @@ public void shouldRequestRetrieval() {
         // move the blob to cold storage
         documentModel = ColdStorageHelper.moveContentToColdStorage(session, documentModel.getRef());
         session.saveDocument(documentModel);
-
+        //Mock AWS Lifecycle rule
+        ColdStorageS3TestHelper.updateObjectClassStorage(session.getDocument(documentModel.getRef()));
```

Comment:
`updateObjectClassStorage` shouldn't be static. You can make it a non static, and create a `ColdStorageS3TestHelper` class instance in the test